### DISCUSSION TITLE
cloud function not running

### DISCUSSION THREAD

**Discussion by upbound-web** [vercel/community#787] [Help] [answered]
Hey guys,
I have an issue with a cloud function not running for me. It work locally on dev server but It wont run on the vercel. The function is an API call to send a text message through twilio. When it runs I get a timeout error from vercel. But I wouldn't imagine that this should cause a timeout. What am I doing wrong? I have a send email route that works fine and it is setup the same. Can anyone help?

**Comment by amyegan** [comment, marked as answer]
_Solution shared in reply https://github.com/vercel/community/discussions/787#discussioncomment-3438829_

Hi @upbound-web! If you can share your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I would be happy to take a closer look. There are a lot of things that could be interfering here, so it's hard to say what the problem is without seeing it for myself.

**Reply by upbound-web** [reply]
Hey thank you so much for your response. I have found the issue. It has 2 res' that was triggering that seemed to cause an issue. I am not sure why it wasn't an issue on local host but they obviously work slightly different.
